### PR TITLE
doc: add meeting minutes CPC 2023-01-03

### PR DESCRIPTION
**Pull request by joesepi**
closes: #989 

Signed-off-by: Joe Sepi <sepi@joesepi.com>

### CODE REVIEW

**Review by ljharb — APPROVED** [review]
so punctual!

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by ovflowd — APPROVED** [review]
I forgot to attend the meeting, but glad to hear it went all well :)

File: meetings/2023/2023-01-03.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  * Still waiting on LF IT staffing
+  * There may be a similar effort at OpenSSF
+
+* Clarify requirements around CoC violation [#786](https://github.com/openjs-foundation/cross-project-council/issues/786)
+  * some of this work may have been done in recent working session
+  * as we move forward with other CoC work, keep an eye on this to make sure we are moving task list forward
+
+## Next week's working session
+
+Next weeks working session: review process issue #982
```
Comment (joesepi) [inline meetings/2023/2023-01-03.md:58]:
```suggestion
Next weeks working session: review process issue #982

EDIT post meeting: will work on Code of Conduct issues in working session next week
```

**Pull request merged**